dbCalls.js

```diff
@@ -0,0 +1,61 @@
+import supabase from "./supabase.js"
+
+export async function getAllDataFromDB() {
```

Comment:
Function expression please

```suggestion
export const getAllDataFromDB = async () {.... }
```